cmd/askd/handlers/form_submission.go

```diff
@@ -253,3 +307,25 @@ func (formSubmissionHandle) RemoveFlag(c *app.Context) error {
 
 	return nil
 }
+
+// encodeCSV gets all the submissions and encode them into a CSV
+func encodeCSV(s []submission.Submission) ([]byte, error) {
+
+	var buf bytes.Buffer
+	w := csv.NewWriter(&buf)
+	if err := w.Write(s[0].GetQuestions()); err != nil {
```

Comment:
I don't think that this would work though. If someone delete a question and wrote a new question, the headers wouldn't work at all, because the "row" for the submission would have answers that wouldn't line up.